mungegithub/README.md

```diff
@@ -33,9 +34,17 @@ As the repo admin or the directory owner, one may approve and merge this PR into
 
   >  Note: Do not comment "/approve" or add "approved" label unless you are 100% sure you want this change, because after you say that, the pr will be merge any minutes.
     
-* **Release-note** Release note enforcement is another feather we are seeking from Mungegithub. With template, when prs are create, people should add release note (can be "None") in the pr description. Depended on the release message left, Mungegithub will add "release-note", "release-note-none". If you leave it empty, "release-note-needed" will be added and is going to block merging.
+* **Release-note** Release note enforcement is another feather we are seeking from Mungegithub. With template, when prs are create, people should add release note (can be the word "none") in the pr description. Depended on the release message left, Mungegithub will add "release-note-none"(if you write string `none`) or "release-note-action-required"(if you write string `action required`), "release-note"(Any other message). If you leave it empty, **"do-not-merge/release-note-label-needed"** will be added and is going to block merging.
```

Comment:
feather?

Reply:
s/is going/will

Reply:
when PRs are created - correct tense, also capitalize PRs everywhere

Reply:
Thanks for pointing out these :)